static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -0,0 +1,134 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+import * as qs from 'query-string';
+
+import {addLoadingMessage} from 'sentry/actionCreators/indicator';
+import {BaseAvatar} from 'sentry/components/core/avatar/baseAvatar';
+import {Button} from 'sentry/components/core/button';
+import type {SelectKey, SelectOption} from 'sentry/components/core/compactSelect';
+import {CompactSelect} from 'sentry/components/core/compactSelect';
+import Form from 'sentry/components/forms/form';
+import FormModel from 'sentry/components/forms/model';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+type Installation = {
+  avatar_url: string;
+  github_organization: string;
+  installation_id: number;
+};
+
+type GithubInstallationProps = {
+  installation_info: Installation[];
+};
+
+export function GithubInstallationSelect({installation_info}: GithubInstallationProps) {
+  const [model] = useState<FormModel>(() => new FormModel());
+  const [installationID, setInstallationID] = useState<SelectKey | null>(null);
```

Comment:
Do we need a model and form if we're going to use react state separately? I'd suggest just scrapping the form/model altogether, we're not using most of it's features and since this is a single field with all the options pre-loaded and a custom submit, it's easier just to do it yourself.